.github/workflows/main.yml

```diff
@@ -0,0 +1,131 @@
+name: CI
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  lint:
+    name: Check lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Cache pip
+        uses: actions/cache@v2
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('requirements.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-
+            ${{ runner.os }}-
+      - name: Install dependencies
+        run: |
+          set -xe
+          python -VV
+          python -m pip install --upgrade tox
+      - name: Check lint
+        run: tox -e check-lint
+
+  tests:
+    name: Python ${{ matrix.python-version }}
+    runs-on: ubuntu-latest
+    needs: lint
+
+    services:
+      postgres:
+        image: postgres:10
+        env:
+          POSTGRES_DB: procrastinate
+          POSTGRES_PASSWORD: password
+        # Set health checks to wait until postgres has started
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+        ports:
+          - 5432:5432
+
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
```

Comment:
Can we add a comment to make sure this stays synced with tox ? (and the other way around)

Reply:
Yes ! Additionaly, I will try to find a way to avoid using `tox-gh-actions`, and to set the matching between tox version with pythons versions in the `main.yml` file. I do not like to put some CI logic in the `tox.ini` file.

Reply:
isn't the TOX_ENV env var trick that was used in Travis applicable here too ?